common/djangoapps/student/tests/test_views.py

```diff
@@ -0,0 +1,56 @@
+"""
+test views
+"""
+import ddt
+from mock import patch
+from django.core.urlresolvers import reverse
+from certificates.models import CertificateStatuses
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+@ddt.ddt
+class TestStudentDashboardUnenrollments(ModuleStoreTestCase):
+    """
+    Test to ensure that the student dashboard does not show the unenroll button for users with certificates.
+    """
+    USERNAME = "Bob"
+    EMAIL = "bob@example.com"
+    PASSWORD = "edx"
+    UNENROLL_ELEMENT = '<li class="actions-item" id="actions-item-unenroll-0">'
```

Comment:
It is a little more fragile than I would like.  If I were using a css selector I would probably only use the `id`.  Unfortunately I don't think pyquery has the right license for us to just use https://github.com/dsc/pyquery/blob/master/LICENSE.txt

We could just search for the `id` rather than the whole element.  I'd hate for this test to break because we added another class to this element.